### PR TITLE
Improve saving release props

### PR DESCRIPTION
**Pull request by radtriste**
This will happen before the decision is taken so that if the input is timeout or any other problem, we still have the file archived

### CODE REVIEW

**Review by mbiarnes — APPROVED** [review]
+LGTM

File: Jenkinsfile.release [hunk 1]
```diff
@@ -64,9 +64,9 @@ pipeline {
                             }
                             def jobMap = getJob(jobName)
                             jobMap.put(jobAttr, value)
-                        }
-                        echo "Previous run jobs config imported: ${getAllJobs()}"
                     }
+                        echo "Previous run jobs config imported: ${getAllJobs()}"
```
Comment (Ginxo) [inline Jenkinsfile.release]:
```suggestion
                    echo "Previous run jobs config imported: ${getAllJobs()}"
```

File: Jenkinsfile.release [hunk 2]
```diff
@@ -454,42 +426,56 @@ pipeline {
...
 
-def registerJob(String jobName, def job) {
-    JOBS[jobName] = job
+def registerJob(String jobName, def job, String decision = '') {
+    def registeredJob = [
+        result: job.result,
+        absoluteUrl: job.absoluteUrl,
+    ]
+    if (decision) {
+        registeredJob.decision = decision
```
Comment (Ginxo) [inline Jenkinsfile.release]:
```suggestion
        registeredJob.decision = decision ?: registeredJob.decision
```
and remove the `if`

**Review by Ginxo — CHANGES_REQUESTED** [review]
One minor and one trivial proposal

File: Jenkinsfile.release (hunk 2 above)
Comment (radtriste) [reply Jenkinsfile.release]: This part was removed

File: Jenkinsfile.release (hunk 1 above)
Comment (radtriste) [reply Jenkinsfile.release]: corrected indentation

**Review by Ginxo — APPROVED** [review]
LGTM

**Pull request merged**